source/Engine.cpp

```diff
@@ -592,6 +592,10 @@ void Engine::Step(bool isActive)
 		info.SetBar("shields", flagship->Shields());
 		info.SetBar("hull", flagship->Hull(), 20.);
 		info.SetBar("disabled hull", min(flagship->Hull(), flagship->DisabledHull()), 20.);
+		if(flagshipHit)
```

Comment:
Single-line loop or conditional bodies don't need braces per [style guide](http://endless-sky.github.io/styleguide/styleguide.xml). (Search for "single-line loop bodies".)